.github/workflows/build.yml

```diff
@@ -118,7 +118,16 @@ jobs:
     - name: Build samples
       run: mvn -B install --file fhir-examples/pom.xml --no-transfer-progress
     - name: Build parent without tests
-      run: mvn -B install --file fhir-parent/pom.xml -DskipTests --no-transfer-progress
+      # As the e2e tests are the fastest, opting to put javadoc:javadoc in fhir-parent
+      # The profile avoids UML diagram creation
```

Comment:
I wonder if it should just be its own separate step.  Or maybe even its own job...

Reply:
I thought about the cognitive complexity of where you would need to go for checks.

Reply:
I'm not sure if it should be it's own step.  It seems like a waste of resources. 
... 
And at the same time, I'm not sure it's a bad idea an atomic flag to a possible failure.